test/rop.jl

```diff
@@ -2,27 +2,27 @@
 @testset "ROP" begin
 
     @testset "test ac rop" begin
-        mn_data = build_mn_data(case5_restoration, replicates=2)
+        mn_data = build_mn_data(case5_restoration, replicates=1)
         @testset "5-bus case" begin
             result = PowerModelsRestoration.run_rop(mn_data, PowerModels.ACPPowerModel, juniper_solver)
 
             @test result["termination_status"] == LOCALLY_SOLVED
-            @test isapprox(result["objective"], 4294.38; atol = 1e-2)
+            @test isapprox(result["objective"], 14716.28; atol = 1e-2)
```

Comment:
Any idea why the objective changes so much?

Reply:
Here, the solution is ~ 3x and expected